src/main/java/org/gradle/profiler/ScenarioLoader.java

```diff
@@ -210,6 +207,7 @@ public ScenarioLoader(GradleBuildConfigurationReader gradleBuildConfigurationRea
                 File bazelHome = getToolHome(executionInstructions);
                 File outputDir = new File(scenarioBaseDir, "bazel");
                 int warmUpCount = getWarmUpCount(settings, scenario);
+                List<BuildMutator> mutators = getMutators(scenario, scenarioName, settings, warmUpCount, buildCount);
```

Comment:
Why do these individual calls here now? Seems redundant.

Reply:
Warmup count and build count are calculated differently in some cases. And since we these two counts to the configurators, we have to handle it that way